Docs/Actions/ReplaceTokensInText.md

```diff
@@ -0,0 +1,163 @@
+# ReplaceTokensInText
+
+## Definition
+
+Replaces placeholders in a body of text with their associated values. Non-recursive.
+
+### Tooling Support
+
+| Environment  | Support |
+| ------------ | ------- |
+| Azure Portal | :white_check_mark: |
+| Core Tools   | :white_check_mark: |
+| VS Code      | :white_check_mark: |
+
+## Parameters
+
+**`type`**  
+**`ReplaceTokensInText`**
```

Comment:
If writeToFile and appendToFile have a property called replaceToken then why  do we need this action?

Reply:
This is covered in the Remarks section of this document; this action is for future template scenarios that may require a token replacement step that isn't paired with a file write.